### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ravi1225**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,14 +13,17 @@
     "@react-navigation/native": "^5.9.2",
     "@react-navigation/stack": "^5.14.2",
     "expo": "~40.0.0",
+    "expo-splash-screen": "~0.8.1",
     "expo-status-bar": "~1.0.3",
+    "link": "^0.1.5",
```
Comment (priyarajputexc) [inline package.json]: remove this. this is of no use

File: package.json [hunk 2]
```diff
@@ -13,14 +13,17 @@
...
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz",
     "react-native-gesture-handler": "~1.8.0",
     "react-native-reanimated": "~1.13.0",
     "react-native-safe-area-context": "3.1.9",
     "react-native-screens": "~2.15.2",
+    "react-native-simple-splash-screen": "^1.0.1",
```
Comment (priyarajputexc) [inline package.json]: remove this. this is of no use

Comment (priyarajputexc) [reply package.json]: do not import any standard-less package.
see it has only 53 downloads. so we don't use such packages - https://www.npmjs.com/package/react-native-simple-splash-screen

File: src/screen/react-nativeconfig.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    assets: ['./assets/Fonts']
+}
```
Comment (priyarajputexc) [inline src/screen/react-nativeconfig.js]: delete this file
as this is not of any use. this way is used in react native app, not expo apps

File: package.json (hunk 2 above)
Comment (ravi1225) [reply package.json]: Ok

**Pull request merged**